### ISSUE TITLE
Fix flashes

### ISSUE THREAD

**Issue by elliotBraem** [NEARBuilders/gateway#25]
After merging the component library #21, I am noticing some red flashes on the UI -- this is typically because a VM.require is not fully loaded before attempting to load it.

The solution is typically to just give a fallback such as:

```
const { Button } = VM.require("/path") || ( () => <></> );
```

This way, when the functional component is called before it is loaded, it resolves successfull to <></>.

## Flashes
- [ ] Go to nearbuilders.org/feed, the left sidebar flashes looking for Button
- [ ] Go to nearbuilders.org/resources, the left sidebar flashes looking for Button
- [ ] Go to nearbuilders.org/library, the page flashes looking for Button

### Note
Since these are all button, I know that in buildhub.near/widget/components, we VM.require button from and buildhub.near/widget/components.Button then return it from the module. It's possible the loading issues comes from here rather than the Widgets that are using Button.

**Comment by elliotBraem** [comment]
I thought this was solved in @itexpert120 #29, but it does not appear so on the current gateway.

Things may act differently when developing locally since local relies on a readily available redirectmap rather than an async request.

Also, I think there was an issue with the advice I gave:

When we do,
```
const { Button } = VM.require("/path");
```

VM.require returns an object where Button is a function, essentially this:

```
const { Button } = {
    Button: () => <button />
}
```

So then a fix like `() => <></>` doesn't actually fix "Button not found".

Something like this would work:

```
const { Button, Post } = VM.require("/path");
Button = Button || () => <></>
Post = Post || () => <></>
```

Or:

```
const { Button, Post } = VM.require("/path") || { Button: () => <></>, Post: () => <></> }
```
